amazon-s3/7.0/modules/ROOT/pages/amazon-s3-connector-reference.adoc

```diff
@@ -341,7 +341,11 @@ Design your application to parse the contents of the response and handle it appr
 | Destination user metadata a| Object |  When copying an object, you can preserve all metadata (default) or specify new metadata. |  |
 | Modified since a| DateTime |  Copies the object only if the object is modified after the specified date. |  |
 | Unmodified since a| DateTime |  Copies the object only if the object is not modified after the specified date. |  |
-| Encryption a| String |  Specifies the appropriated encryption header. |  |
```

Comment:
is Encryption field supposed to be deleted?

Reply:
Yes. Confirmed with the dev team. Encryption is replaced with Server Side Encryption.